tests/cases/fourslash/codeFixSpellingPrivateNameInIn.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path='fourslash.ts' />
+
+////class A {
+////   #foo: number;
+////   static isA(v: any) {
```

Comment:
the two codefix tests will have to change to `v: A` without the lexical lookup

Reply:
done :)